specs/009-add-scoring/plan.md

```
@@ -26,7 +26,7 @@ Score persists across levels within a game session and updates in real-time on t
 ## Constitution Check
 
 *GATE: Must pass before Phase 0 research.*
-                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 *Re-check after Phase 1 design.*
+                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             *Re-check after Phase 1 design.*
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)